src/components/policy/src/policy/src/policy_manager_impl.cc

```diff
@@ -1131,6 +1133,8 @@ void PolicyManagerImpl::OnChangedRemoteControl(
   if (!access_remote_->IsEnabled()) {
     std::string default_hmi;
     if (GetDefaultHmi(application_id, &default_hmi)) {
+      Subject who = { device_id, application_id };
```

Comment:
Why is this code appear after check for default HMI? It seems independent on default HMI level?